File: 1-multiples_of_3_or_5/solution.cpp
#include <iostream>

int solution()
{
    int c { 0 }; // variable to store the sum
    int r { 1000 }; // upper bound
    for (int i = 1; i < r; ++i) {
        if (i % 3 == 0 || i % 5 == 0) {
            c += i;
        }
    }

    return c;
}

int main(int argc, char **argv)
{
    std::cout << solution() << "\n";
    return 0;
}

File: 1-multiples_of_3_or_5/solution.py
def solution():
    c = 0               # variable to store the sum
    r = 1000            # upper bound
    for i in range(1, r):
        if i % 3 == 0 or i % 5 == 0:
            c += i

    return c

if __name__ == "__main__":
    print(solution())

